rclpy/executors/examples_executors/callback_group.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from examples_executors.listener import Listener
+import rclpy
+from std_msgs.msg import String
+
+
+class DoubleTalker(rclpy.Node):
+    def __init__(self):
+        super().__init__("double_talker")
```

Comment:
Please use single quotes (if that doesn't imply a need for escaping other single quotes within the string).